.gitignore

```diff
@@ -36,6 +36,7 @@
 /docs/source/__pycache__/
 /docs/source/plugins/
 /elisp/.cask/
+/test/testdata/*.exe
```

Comment:
What exe files are generated? Out of interest, not really a review comment.

Reply:
./test/testdata/FileWithWarning.exe
./test/testdata/HareRename.exe